api/src/main/java/marquez/db/DatasetFacetsDao.java

```diff
@@ -126,9 +126,9 @@ void insertDatasetFacet(
   default void insertDatasetFacetsFor(
       @NonNull UUID datasetUuid,
       @NonNull UUID datasetVersionUuid,
-      @NonNull UUID runUuid,
+      UUID runUuid,
       @NonNull Instant lineageEventTime,
-      @NonNull String lineageEventType,
+      String lineageEventType,
```

Comment:
Can we use `DATASET` or `JOB` as  the `lineageEventType`? I think it would be helpful to know the event type.

Reply:
Currently, this column holds run states and its name is kind of misleading. Its name fits static lineage scenario well. However, I don't think we should store in a single column run-state and event-type. Renaming this column would require significant amount of work witch changes all over the project, including the spec.

Assuming this column contains run state update, then keeping it to `null` for `DatasetEvent` makes sense to me.

Reply:
I agree it's misleading, but Marquez doesn't necessarily have to adhere exactly to OpenLineage concepts and can be redefine them. We can set lineage_event_type` as `null `, but would prefer we (eventually) remap `eventType` to `runState`.